Dockerfile

```diff
@@ -1,56 +1,53 @@
 # syntax=docker/dockerfile:1.4
 
 ####################################################################################################
-## Prepare Workspace
+## Build Packages
```

Comment:
There's not really any meaningful separation between what we've called "Workspace" and "Pruned," and there's some meaningful layer combinations we can do between them.

Reply:
Fair enough!